### PR TITLE
Fixing link to documentation

### PR DESCRIPTION
**Pull request by stephenpurkiss**

### CODE REVIEW

**Comment by steveej** [comment]
We've set up a redirection at https://docs.holochain.love so we could potentially reintroduce separate documentation at this URL. Would you favor the direct link here despite the redirection?

**Comment by stephenpurkiss** [comment]
Hi @steveeJ! I prefer consistency and not getting confused ;) So I just looked at the link and what it goes to again and I’ve seen references to “holonix”, “Holochain Developer Tools”, “holochain.love”.

From my quick look there is nothing saying “The Holochain Developer Tools, holonix,”, only “install the Holochain Developer Tools.” heading then it says this is how to install holonix which I presume is the developer tools then the url is Holochain.love.

Are you confused yet? ;)

If it is a distinct enough area from the rest of the system then sure, create another space however I would’ve thought that would be more troublesome to maintain than all in one place hence why I didn’t question why the documentation had been merged and instead created this issue because one thing I learned was when I see something I think is a small issue and a quick fix like I thought this was I create a pull request because often devs are busy doing what they’re doing and don’t really see little stuff like this and hence my pull request.

However then I wake up and see your reply and I am pre-coffee so you’re getting a long rant as I literally say what I see.

My hunch is KISS - keep it simple st... ;) Choosing one of the three is hard though. “Love” - well we all love the Holochain, that doesn’t say “Dev tools” and I guess someone somewhere might find an issue using the word “love” cos that seems to be the thing these days so maybe better using that for garnering sponsorship, donations, etc. Or maybe I’m thinking way too much into it, I just think there’s a lot more ways to love Holochain than just code contributions...

Holonix is good if it refers to an environment, which I am gathering the way nix works it is because it’s the development environment for Holochain, and it’s using nix. So that is perfectly sensible for using, and if you want it on a separate url then I would use docs.holonix.something not Holochain.love.

Holochain Developer Tools is well pretty boring but to the point, if you can’t focus on using holonix because I’m wrong in my assumptions in the paragraph above then just be boring at the moment to make it easy for people to get on with doing what they’re doing which is finding documentation for the Holochain developer tools.

so, in summary, my vote is for calling the Holochain developer tools holonix (which I think they are already but within this small scope I haven’t seen that defined) and having a domain which reflects that for the docs. holonix.holochian.org? “Welcome to Holonix, the Developer Tools for Holochain!”

I will be interested to see what my brain thinks after some Dark Arts Coffee “Raise the Dead” or “Shake Me Lucifer” lol https://bit.ly/darkartscoffee

HTH

sbp

**Comment by steveej** [comment]
good morning @stephenbpurkiss :smile:

**Review by steveej — APPROVED** [review]

**Comment by stephenpurkiss** [comment]
"holochain.dev" is available, recommend securing, could use for this.

I use Namecheap, is $14.98 there. Cheapest seems to be porkbun $11.65 - not heard of or used them but cursory google returns ok results.
